x-pack/plugins/cloud_security_posture/common/schemas/csp_rule_template.ts

```diff
@@ -5,24 +5,34 @@
  * 2.0.
  */
 import { schema as rt, TypeOf } from '@kbn/config-schema';
+import { cspRuleMetadataSchema } from './csp_rule_metadata';
 
-const cspRuleTemplateSchema = rt.object({
+export const cspRuleTemplateSchemaV830 = rt.object({
```

Comment:
We talked about moving the schemas to the server directory as they use `@kbn/config-schema` - consider doing that in the scope of this PR

Reply:
will do